webpack/JobWizard/__tests__/JobWizard.test.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
 import * as patternfly from '@patternfly/react-core';
-import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
+import { testComponentSnapshotsWithFixtures, mount } from '@theforeman/test';
 import JobWizardPage from '../index';
 import { JobWizard } from '../JobWizard';
 
 jest.spyOn(patternfly, 'Wizard');
-patternfly.Wizard.mockImplementation(props => <div>{props}</div>);
+patternfly.Wizard.mockImplementation(props => <div>{props.navAriaLabel}</div>);
+const mockStore = configureMockStore();
+const store = mockStore({});
 const fixtures = {
   'renders ': {},
 };
-describe('JobWizardPage', () => {
-  describe('rendering', () =>
-    testComponentSnapshotsWithFixtures(JobWizardPage, fixtures));
-});
+describe('JobWizardPage rendering', () =>
+  testComponentSnapshotsWithFixtures(JobWizardPage, fixtures));
 
-describe('JobWizard', () => {
-  describe('rendering', () =>
-    testComponentSnapshotsWithFixtures(JobWizard, fixtures));
+describe('JobWizard rendering', () => {
+  describe('render', () => {
+    const component = mount(
+      <Provider store={store}>
+        <JobWizard />
+      </Provider>
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
how about instead of a snapshot, you can test if the tabs exist and try to jump between them?

Reply:
Added test in the integration test